x-pack/test/detection_engine_api_integration/utils.ts

```diff
@@ -360,9 +360,12 @@ export const deleteSignalsIndex = async (
  * for testing uploads.
  * @param ruleIds Array of strings of rule_ids
  */
-export const getSimpleRuleAsNdjson = (ruleIds: string[]): Buffer => {
+export const getSimpleRuleAsNdjson = (ruleIds: string[], enabled?: boolean): Buffer => {
   const stringOfRules = ruleIds.map((ruleId) => {
     const simpleRule = getSimpleRule(ruleId);
+    if (enabled != null) {
+      return JSON.stringify({ enabled, ...simpleRule });
+    }
     return JSON.stringify(simpleRule);
   });
   return Buffer.from(stringOfRules.join('\n'));
```

Comment:
Optionally you can just use defaulted parameters like so here and at the `getSimpleRule` level and it should make things easier:

```ts
/**
 * Given an array of rule_id strings this will return a ndjson buffer which is useful
 * for testing uploads.
 * @param ruleIds Array of strings of rule_ids
 * @param enabled Enables the rule on creation or not. Defaulted to false to enable it on import
 */
export const getSimpleRuleAsNdjson = (ruleIds: string[], enabled = true): Buffer => {
  const stringOfRules = ruleIds.map((ruleId) => {
    const simpleRule = getSimpleRule(ruleId, enabled);
    return JSON.stringify(simpleRule);
  });
  return Buffer.from(stringOfRules.join('\n'));
};
```

```ts
/**
 * This is a typical simple rule for testing that is easy for most basic testing
 * @param ruleId
 *  @param enabled Enables the rule on creation or not. Defaulted to false to enable it on import
 */
export const getSimpleRule = (ruleId = 'rule-1', enabled = true): CreateRulesSchema => ({
  name: 'Simple Rule Query',
  description: 'Simple Rule Query',
  enabled,
  risk_score: 1,
  rule_id: ruleId,
  severity: 'high',
  index: ['auditbeat-*'],
  type: 'query',
  query: 'user.name: root or user.name: admin',
});
```

Reply:
Ok awesome I will update this. Thanks!